collect_app/src/main/java/org/odk/collect/android/tasks/DownloadFormListTask.java

```diff
@@ -405,24 +405,22 @@ private boolean isNewerFormVersionAvailable(String md5Hash) {
     }
 
     private boolean areNewerMediaFilesAvailable(String formId, String formVersion, List<MediaFile> newMediaFiles) {
-        if (newMediaFiles != null) {
```

Comment:
I just removed a double null check since we check it here https://github.com/opendatakit/collect/pull/1738/files#diff-10d77d66d8d1d9f9edc5c6e5c1706489R236

Reply:
I would have tended to keep the null-check in the method. This requires all method clients to remember it or to suffer a crash. A `@NotNull` annotation could help.